packages/teleterm/src/ui/ClusterConnect/ClusterLogin/FormLogin/PromptWebauthn/PromptWebauthn.tsx

```diff
@@ -0,0 +1,202 @@
+/**
+ * Copyright 2021-2022 Gravitational, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import { Box, ButtonSecondary, ButtonPrimary, Text, Image, Flex } from 'design';
+import FieldInput from 'shared/components/FieldInput';
+import Validation from 'shared/components/Validation';
+import LinearProgress from 'teleterm/ui/components/LinearProgress';
+import type { WebauthnLogin } from '../../useClusterLogin';
+
+const svg = require('./hardware.svg');
+
+export function PromptWebauthn(props: Props) {
```

Comment:
yeah... it was because we had two types:

- passwordless (hardware only) 
- username + second factor 

and i'm re-using the `prompt` type `tap` to refer to both cases, so I tried to use `webauthn` where make sense

Reply:
Ah, fine, I forgot about the second factor, it makes sense now.